tfexec/apply_test.go

```diff
@@ -65,3 +65,63 @@ func TestApplyCmd(t *testing.T) {
 		}, nil, applyCmd)
 	})
 }
+
+func TestApplyJSONCmd(t *testing.T) {
```

Comment:
There's also the integration testing folder if you want to add end-to-end testing code. 👍

Reply:
Have added e2e tests.